plugins/modules/ssh_config.py

```diff
@@ -274,6 +282,9 @@ def ensure_state(self):
             controlpersist=fix_bool_str(self.params.get('controlpersist')),
             dynamicforward=self.params.get('dynamicforward'),
         )
+        if self.params.get('other_options'):
+            for key, value in self.params.get('other_options').items():
+                args[key] = str(value)
```

Comment:
Instead of converting values to strings, fail the module if a value is not a string.

(Also use `string_types` to check for strings, not `str`; do `from ansible.module_utils.six import string_types` for that.)

Reply:
Have added a check for string_types verification for values causing a module fail if the test fails, and removed the str conversion of values